java-build/entrypoint.sh

```diff
@@ -4,6 +4,13 @@ set -eu
 
 accessToken=$1
 projectPath=$2
+if [ $3 = "true" ]; then
```

Comment:
`minor`
Is it possible to do this in one line, or do we need the explicit if/else structure?

Reply:
It should work with direct assignment